src/client/pages/OfferNew/Introduction/Sidebar/StartDate.tsx

```diff
@@ -30,28 +30,35 @@ import {
   isBundle,
 } from 'pages/OfferNew/utils'
 import { useTextKeys } from 'utils/textKeys'
+import { Size } from 'components/types'
 import { gqlDateFormat } from './utils'
 
 interface Props {
```

Comment:
Nitpick but these props are use at line 355
export const StartDate: React.FC<Props> = ({
Maybe we can move it just above the Component definition, IMO it makes it easier to navigate the file.

Reply:
Or move the other component to its own file maybe? Or is that too much work now? 🤔

Reply:
I think it's a good idea to split it into two files 🙌 But it turned out it was a bit more than a minor fix so I'd rather do it outside this PR if it's ok with you :)